src/screens/FeedScreen/Header.js

```diff
@@ -139,6 +139,7 @@ const _renderAnonimity = ({
                 karmaScore={karmaScore}
                 anonUserInfo={anonUserInfo}
                 version={version}
+                withKarma
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `withKarma` prop is being passed to the `<Anonimity />` component without a value. If it's meant to be a boolean, it should be explicitly set to `true` or `false`. If it's supposed to receive a variable or function, that should be provided. Here's an example assuming it's a boolean:

```diff
-                withKarma
+                withKarma={true}
```
If it's not a boolean, replace `true` with the appropriate value or function.


<!-- This is an auto-generated comment by OSS CodeRabbit -->